NGitLab/Models/MergeRequestApprovals.cs

```diff
@@ -7,6 +7,12 @@ public class MergeRequestApprovals
         [JsonPropertyName("approvers")]
         public MergeRequestApprover[] Approvers;
 
+        [JsonPropertyName("approved_by")]
+        public MergeRequestApprover[] ApprovedBy { get; set; }
```

Comment:
You need to add this to the public API definition. You can use Visual Studio's "Quick Actions and Refactorings..." to quickly do it for all additions:

![image](https://user-images.githubusercontent.com/38251605/173063043-68ae55e9-fda7-427f-9f5c-bb3a531f1e08.png)

Reply:
Updated, thanks.